pallets/payment/src/benchmarking.rs

```diff
@@ -0,0 +1,76 @@
+use super::*;
+
+use crate::Pallet as Payment;
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite, whitelisted_caller};
+use frame_system::RawOrigin;
+use orml_traits::MultiCurrency;
+use virto_primitives::PaymentState;
+
+const SEED: u32 = 0;
+
+fn setup_caller<T: Config>() -> T::AccountId {
+	let caller = whitelisted_caller();
+	let value: BalanceOf<T> = 10u32.into();
+	let asset: AssetIdOf<T> = "test".into(); // TODO : figure out way to convert to type of CurrencyId of orml-tokens
```

Comment:
@olanod Not sure how to solve this 🤔 [CurrencyId trait](https://github.com/open-web3-stack/open-runtime-module-library/blob/17a791edf431d7d7aee1ea3dfaeeb7bc21944301/tokens/src/lib.rs#L188) is not very helpful. Acala seems to be [tightly coupling the CurrencyId](https://github.com/AcalaNetwork/Acala/blob/b2054d585665478422e9d14c0c8620dc832f708d/modules/homa-lite/src/lib.rs#L75), looks like a bad design for us with multiple instances.

Reply:
Still not sure what the problem is, does it fail with `"test"`? if so, maybe the string literal simply doesn't implement the required traits. What about using one of the `Asset` variants?

Reply:
The problem is I'm not sure how to convert any data to the "type" CurrencyId. When I use Asset, the error is 

```
note: expected associated type `<<T as pallet::Config>::Asset as MultiCurrency<<T as frame_system::Config>::AccountId>>::CurrencyId`
                         found enum `Asset`
```

But the [CurrencyId](https://github.com/open-web3-stack/open-runtime-module-library/blob/17a791edf431d7d7aee1ea3dfaeeb7bc21944301/tokens/src/lib.rs#L188) does not require anything that allows conversion. So the challenge is to convince the compiler to accept our mock data as CurrencyId 🤔